src/mesh/vtk_helper.jl

```diff
@@ -115,6 +119,96 @@ function quad_vtk_order(corner_verts, order, dim, skip = false)
     return coords
 end
 
+"""
+    wedge_vtk_order(corner_verts, order, dim)
+
+Compute the coordinates of a VTK_LAGRANGE_wedge of a wedge of order order 
+defined by the coordinates of the vertices given in corner_verts. dim is the
+dimension of the coordinates given. If skip is set to true, the coordinates
+of the vertex- and edge-points aren't computed, which can be used to compute
+points of a VTK_LAGRANGE_WEDGE
```

Comment:
```suggestion
Compute the coordinates of a `VTK_LAGRANGE_wedge` of a wedge of order `order` 
defined by the coordinates of the vertices given in `corner_verts`. `dim` is the
dimension of the coordinates given. If `skip` is set to `true`, the coordinates
of the vertex- and edge-points aren't computed, which can be used to compute
points of a `VTK_LAGRANGE_WEDGE`
```
The last part "which can be used to compute points of a `VTK_LAGRANGE_WEDGE`" is a little unclear. When would you want to set `skip=true`?

Reply:
For wedges never, I removed this part from the documentation